jina/serve/instrumentation/__init__.py

```diff
@@ -1,6 +1,7 @@
 from typing import Optional
 
```

Comment:
remember to hide the `opentelemetry` imports if possible as `prometheus-client` was doing